style-guide/Gemfile

```diff
@@ -15,6 +15,7 @@ gem "middleman-livereload"
 gem "middleman-minify-html"
 gem 'middleman-s3_sync'
 gem 'fog'
+gem 'xmlrpc'
```

Comment:
Minor: I like to put a comment on lines like this to justify a dependency that would otherwise seem whacko. I actually don't know _what_ tooling depends on `xmlrpc` and `webrick`, but if you do then a little breadcrumb here may clarify that and make it obvious when it can be eliminated in the future.